File: 전효정/2주차/1065.c
n,count=input(),0

def len3(n):
    global count
    count=99
    for i in range(100, int(n)+1):
        i=str(i)
        if int(i[1])==(int(i[0])+int(i[2]))/2:
            count+=1
        else:
            continue
    return count

if len(n)==1 or len(n)==2:
    for i in range(int(n)):
        count+=1
else:
    len3(n)

print(count)

File: 김진호/2주차/1065.py
N = int(input())

res = 0
for i in range(1, N + 1):
    num = list(map(int, str(i)))
    if len(num) == 1:
        res += 1
    else:
        d = num[1] - num[0]
        flag = True
        for j in range(len(num) - 1):
            if num[j + 1] - num[j] != d:
                flag = False
                break
        if flag:
            res += 1
print(res)